extensions/amp-mathml/1.0/storybook/Basic.js

```diff
@@ -0,0 +1,49 @@
+import * as Preact from '#preact';
+
+import {BentoMathml} from '../component';
+
+const formulas = {
+  quadraticFormula: '\\[x = {-b \\pm \\sqrt{b^2-4ac} \\over 2a}.\\]',
+  cauchyFormula: '\\[f(a) = \\frac{1}{2\\pi i} \\oint\\frac{f(z)}{z-a}dz\\]',
+  doubleAngleFormula: '$$ \\cos(θ+φ)=\\cos(θ)\\cos(φ)−\\sin(θ)\\sin(φ) $$',
+};
+
+export default {
+  title: 'Mathml',
+  component: BentoMathml,
+  args: {
+    formula: {
+      quadratic: formulas.quadraticFormula,
+      cauchy: formulas.cauchyFormula,
+      doubleAngle: formulas.doubleAngleFormula,
+    },
+  },
```

Comment:
```suggestion
    style: {
      width: '100%',
      height: 40,
    },
  },
```

Reply:
> 2. The formulas seem to sporadically take a little while (5s) to show up. For instance, the default Preact storybook displays 5 formulas and 2 or 3 show up quickly and the others take a while to emit their `embed-size` event, which is what reveals them. This doesn't happen in the legacy AMP component example (which also seems to use the `embed-size` event to set dimensions), so is this an issue with my use of Preact?

Are you seeing this is still the case when predefining the component dimensions? Try `style={{width: '100%', height: 40}}`.

Reply:
nope, giving it a fixed height shows that the formula is rendered far ahead of when the the `embed-size` event is received and handled. Setting a fixed height would definitely be one solution here, but my concern was more that this wasn't even necessary with the legacy component. So I wasn't sure if there was something slowing down the dispatch/handling of the `embed-size` event.

Reply:
I'm seeing that the event is fired about ~3s into page load for both versions, can you provide more info on the latency discrepancy you're noticing?

Reply:
i think this was a local issue, but i don't see it anymore because of the intersection observer logic